tests/parsers/test_mkdocs.py

```diff
@@ -0,0 +1,51 @@
+from visivo.parsers.mkdocs import Mkdocs
+import yaml
+
+TEST_MKDOCS_OBJECT = {
+        'nav': [
+            {'getting started': 'index.md'},
+            {'including': 'index.md'},
+            {'reference': [
+                {'Configuration': {'trace': [], 'chart': []}},
+                {'cli': 'index.md'}
+            ]}
+        ],
+        'extra_css': [{'Configuration':'something'}]
+    }
+
+mkdocs = Mkdocs()
+
+def test_nav_configuration():
+    nav_config = mkdocs.get_nav_configuration()
+    assert len(nav_config) > 5
+
+def test_docs_content_from_pydantic_model():
+    trace_md = mkdocs.get_md_content('Trace')
+    assert len(trace_md) > 50 
+
+    chart_md = mkdocs.get_md_content('Chart')
+    assert len(chart_md) > 50
+
+    dashboard_md = mkdocs.get_md_content('Dashboard')
+    assert len(dashboard_md) > 50
+
+    target_md = mkdocs.get_md_content('Target')
+    assert len(target_md) > 50
+
+def test_docs_content_from_traceprops_model():
+    bar_md = mkdocs.get_md_content('Bar')
+    assert len(bar_md) > 50 
```

Comment:
Are there any specific assertions that can be made?

Reply:
I'm not sure. Maybe there's a markdown package that we can use to validate elements are building correctly. For example the attribute tables are rendering properly.